StructureDefinition/NHSDigital-Claim.StructureDefinition.xml

```diff
@@ -566,7 +563,7 @@
         <rules value="open" />
       </slicing>
       <short value="Dispensing Endorsement and Charge Payment" />
-      <definition value="Dispensing Endorsement  - Details of the endorsement(s) to support the claim for this medication item. &#xD;&#xA;&#xD;&#xA;Charge Payment - Details on whether a prescription charge was paid for the medication treatment.&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&#xD;&#xA;&quot;programCode&quot;:  [&#xD;&#xA;    {&#xD;&#xA;        &quot;coding&quot;:  [&#xD;&#xA;            {&#xD;&#xA;                &quot;system&quot;: &quot;https://fhir.nhs.uk/CodeSystem/DM-prescription-charge&quot;,&#xD;&#xA;                &quot;code&quot;: &quot;paid-once&quot;,&#xD;&#xA;                &quot;display&quot;: &quot;Paid Once&quot;&#xD;&#xA;            }&#xD;&#xA;        ]&#xD;&#xA;    },&#xD;&#xA;    {&#xD;&#xA;        &quot;coding&quot;:  [&#xD;&#xA;            {&#xD;&#xA;                &quot;system&quot;: &quot;https://fhir.nhs.uk/CodeSystem/medicationdispense-endorsement&quot;,&#xD;&#xA;                &quot;code&quot;: &quot;IP&quot;,&#xD;&#xA;                &quot;display&quot;: &quot;Invoice Price&quot;&#xD;&#xA;            }&#xD;&#xA;        ]&#xD;&#xA;    }&#xD;&#xA;],&#xD;&#xA;&#xD;&#xA;``` &#xD;&#xA;If an endorsement code of PD is used, the number of packaged doses dispensed minus the number of dispenses must be entered in the coding.text field e.g. PD4 for five doses packaged separately and dispensed on one occasion" />
+      <definition value="Dispensing Endorsement  - Details of the endorsement(s) to support the claim for this medication item. &#xD;&#xA;&#xD;&#xA;Charge Payment - Details on whether a prescription charge was paid for the medication treatment.&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&#xD;&#xA;&quot;programCode&quot;:  [&#xD;&#xA;    {&#xD;&#xA;        &quot;coding&quot;:  [&#xD;&#xA;            {&#xD;&#xA;                &quot;system&quot;: &quot;https://fhir.nhs.uk/CodeSystem/DM-prescription-charge&quot;,&#xD;&#xA;                &quot;code&quot;: &quot;paid-once&quot;,&#xD;&#xA;                &quot;display&quot;: &quot;Paid Once&quot;&#xD;&#xA;            }&#xD;&#xA;        ]&#xD;&#xA;    },&#xD;&#xA;    {&#xD;&#xA;        &quot;coding&quot;:  [&#xD;&#xA;            {&#xD;&#xA;                &quot;system&quot;: &quot;https://fhir.nhs.uk/CodeSystem/medicationdispense-endorsement&quot;,&#xD;&#xA;                &quot;code&quot;: &quot;IP&quot;,&#xD;&#xA;                &quot;display&quot;: &quot;Invoice Price&quot;&#xD;&#xA;            }&#xD;&#xA;        ]&#xD;&#xA;    }&#xD;&#xA;],&#xD;&#xA;&#xD;&#xA;```" />
```

Comment:
Where do would you like to see guidance on additional required information required on dispensing endorsements? This was to make the requirement clear to dispensers using the IG